news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,26 @@
+
+"**Medal Hunter**" is one of [unique user titles](/wiki/People/Users_with_unique_titles), which is given for being the first player to unlock an extremely hard to solve Hush-Hush medal! Also [previous "**Medal Hunter**" title](https://osu.ppy.sh/home/news/2022-09-09-how-zetera-became-a-medal-hunter) has been given out nearly **6 years ago**, which definitely made it even more interesting.
+
+In order to obtain such a desirable title, you had to be the first player to find the solution for **Internment** medal and unlock it. **Internment** is a Hush-Hush medal added in the last batch, being so hint-based that developers found unlocking it worth a title. And while only one player could get a shiny "**Medal Hunter**" text under their username, many still have decided to team up with others on [osu! Medal Hunters](https://discord.gg/8qpNTs6) [Discord](https://discord.com) server! 
+
+Medal hunters were trying their best to hunt the solution, but as insanity approached and they have tried an enormous number of things that could've been it, the hope was slowly fading away. On this journey of basking in the shade of an ailing mind, the community had received various hints, including a video recorded by one of medal hunters at [COE](/wiki/Community/cavoe's_osu!_event), featuring [peppy](https://osu.ppy.sh/users/2) himself! 
+
+Within the first few days of the medal releasing, medal hunters were very quick to piece together that the hues were hinting at RGB to some capacity, but nobody knew in what way. From there, medal hunters set out to find how to input various RGB values in game, but nothing seemed to be working. In the worst of medal hunters' struggles, about half of them entering week 2 went full Sherlock Holmes and schizo mode, relying more on insanity than logic, drowning the threads with abominations of theories, some being genuinely good that Ephemeral would take note of, and others... well, let's just say Pac-Man theory. 
+
+But, despite all the challenges the community was still determined to find the solution. One of them was [UberFazz](https://osu.ppy.sh/users/8646059), who was going back and forth, seeking for the way to get this medal. At first they assumed it would be achieving 16777216 score, because that's how many possible colors there are in RGB. Another guess of theirs was to interact with osu! website to find a hidden message somewhere on it, which would then lead to a solution. They were also told by one of developers they had insanely close guesses multiple times, but that was just not enough...
+
+But suddenly, the very last hint was dropped, and after reading it, UberFazz figured exactly what they need to do. Shortly after trying their idea, UberFazz successfully got **Internment** medal and received the "**Medal Hunter**" title, after more than a month since medal release!

```

Comment:
feels like a really abrupt ending

Reply:
@0x84f I've added the ending that should sound better

Reply:
looks good